src/utils/IntlUtils.js

```diff
@@ -90,6 +90,32 @@ const IntlUtils = {
 
         break;
 
+        // portugues
+        case "pt":
+
+          if (!hasIntl) {
+
+            require.ensure([
+                "intl/locale-data/jsonp/pt",
+                "react-intl/dist/locale-data/pt"
+              ], (require) => {
+
+                require("intl/locale-data/jsonp/pt");
```

Comment:
@saitodisse don't forget to require also `react-intl/dist/locale-data/pt` after this line (already included in `react.ensure` array), otherwise the "time ago" thing would not work client-side ☝️